src/map.h

```diff
@@ -51,6 +51,8 @@ typedef std::vector< std::pair< item*, int > > itemslice;
 typedef std::string items_location;
 struct vehicle_prototype;
 using vproto_id = string_id<vehicle_prototype>;
+struct VehicleGroup;
```

Comment:
```
ccache clang++ -DRELEASE -DLOCALIZE -Os -Wall -Wextra   -Werror --std=c++11 -MMD -I/usr/include/lua5.2     -DLUA -I/usr/include/ncursesw -c src/init.cpp -o obj/init.o
In file included from src/init.cpp:15:
src/vehicle_factory.h:14:1: error: 'VehicleGroup' defined as a class here but previously declared as a struct [-Werror,-Wmismatched-tags]
class VehicleGroup {
^
src/map.h:54:1: note: did you mean class here?
struct VehicleGroup;
^~~~~~
class
1 error generated.
make: *** [obj/init.o] Error 1
```

-- [Jenkins Linux/Curses build 3233](http://ci.narc.ro/job/Cataclysm-Matrix/Graphics=Curses,Platform=Linux/3233/console)

Notably, the g++ builds don't seem to care about this -- only clang++ noticed.